frontend/package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
     "formik": "^2.2.9",
+    "history": "^5.3.0",
```

Comment:
let's use history from react-router

https://reactrouter.com/docs/en/v6/hooks/use-navigate

Reply:
yeah, I checked that and noted that we could not use it outside component (inside axios interceptor), I will try find another solution than :)